thetis/options.py

```diff
@@ -881,12 +881,12 @@ class ModelOptions2d(CommonModelOptions):
 
         Note this is only relevant if `use_automatic_wetting_and_drying_alpha` is set to ``True``.
         """).tag(config=True)
-    tidal_turbine_farms = Dict(trait=TidalTurbineFarmOptions(),
-                               default_value={}, help='Dictionary mapping subdomain ids to the options of the corresponding farm')
+    tidal_turbine_farms = Dict(trait=List(TidalTurbineFarmOptions()),
+                               default_value={}, help='Dictionary mapping subdomain ids to the options of the corresponding farm(s)')
```

Comment:
```suggestion
                               default_value={}, help='Dictionary mapping subdomain ids to a list of TidalTurbineFarmOptions instances corresponding
                               to one or more farms.')
```

Reply:
I suppose you could have different densities for different turbine characteristics?

Reply:
I've just done that out of consistency more than anything, happy to go with whatever is preferred!